Cargo.toml

```diff
@@ -76,7 +76,7 @@ topiary-web-tree-sitter-sys = { version = "0.6.1", path = "./topiary-web-tree-si
 topiary-tree-sitter-facade = { version = "0.6.1", path = "./topiary-tree-sitter-facade" }
 topiary-core = { version = "0.6.1", path = "./topiary-core" }
 topiary-config = { version = "0.6.1", path = "./topiary-config" }
```

Comment:
I think `topiary-config` should also change to `0.6.2`. None of the Rust code has changed, of course, but the configuration is probably the most important things atm.